[2024-05-18T19:32:23.987+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:32:23.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-06-19T01:00:00+00:00 [queued]>
[2024-05-18T19:32:24.005+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-06-19T01:00:00+00:00 [queued]>
[2024-05-18T19:32:24.006+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-05-18T19:32:24.016+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2023-06-19 01:00:00+00:00
[2024-05-18T19:32:24.022+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1556) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:32:24.023+0000] {standard_task_runner.py:63} INFO - Started process 1558 to run task
[2024-05-18T19:32:24.023+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2023-06-19T01:00:00+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmp6ccueugi']
[2024-05-18T19:32:24.025+0000] {standard_task_runner.py:91} INFO - Job 328: Subtask hello_task
[2024-05-18T19:32:24.062+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2023-06-19T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:32:24.135+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-19T01:00:00+00:00'
[2024-05-18T19:32:24.137+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:32:24.138+0000] {logging_mixin.py:188} INFO - 2023-06-19/
[2024-05-18T19:32:24.139+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:32:24.140+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:32:24.148+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2023-06-19T01:00:00+00:00, execution_date=20230619T010000, start_date=20240518T193224, end_date=20240518T193224
[2024-05-18T19:32:24.198+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:32:24.217+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:32:24.220+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
